rest-service/manager_rest/rest/resources_v1/events.py

```diff
@@ -128,37 +194,61 @@ def _build_select_query(_include, filters, pagination, sort):
                 .filter(
                     Log._execution_fk == Execution._storage_id,
                     Execution._deployment_fk == Deployment._storage_id,
-                    Execution.id == bindparam('execution_id'),
                 )
             )
+            logs_query = Events._apply_range_filters(
+                logs_query, Log, range_filters)
+            if 'deployment_id' in filters:
+                logs_query = logs_query.filter(
+                    Deployment.id.in_(filters['deployment_id']))
 
-        if '@timestamp' not in sort or sort['@timestamp'] != 'asc':
-            raise manager_exceptions.BadParametersError(
-                'Sorting ascending by `timestamp` is expected')
+            query = query.union(logs_query)
+
+        if 'execution_id' in filters:
+            query = query.filter(Execution.id == bindparam('execution_id'))
+
+        if '@timestamp' in sort:
```

Comment:
What I've seen in the integration test cases and in the CLI is just the use of `@timesamp`. However, it makes sense to support `timestamp` to allow and deprecate `@timestamp`.